README.md

```diff
@@ -1 +1,36 @@
-# ES-Awesome-book
\ No newline at end of file
+# ES-Awesome-book
+
+ES6-Awesome-book
+In this project I made a website that stores books in a collection and dynamcally displays them at the top of the page.
+
+Built With
+HTML
+CSS
+JavaScript
+Getting Started
+To obtain a local copy of the project follow this steps:
+
+Open your console.
+Type the following "git clone https://github.com/ijeomaodinko/Es-Awesome-book.git" if you are using HTTPS or "git clone git@github.com:ijeomaodinko/E6-Awesome-book.git" if you are using a SSH key.
+Press ENTER to create your local clone.
+After creating it, you can open the "index.html" file in your browser.
+
+Author
+Ijeoma Odinko
+
+GitHub: @ijeomaodinko
+Twitter: @iodinko
+LinkedIn: @ijeomaodinko
```

Comment:
- [ ] [OPTIONAL] You might want to use some of the styling used in [the template](https://github.com/microverseinc/readme-template) so it will be easier to read for any user.